progenitor-impl/src/method.rs

```diff
@@ -1655,6 +1675,100 @@ impl Generator {
 
         impl_body
     }
+
+    fn get_body_param(
+        &mut self,
+        operation: &openapiv3::Operation,
+        components: &Option<Components>,
+    ) -> Result<Option<OperationParameter>> {
+        let body = match &operation.request_body {
+            Some(body) => body.item(components)?,
+            None => return Ok(None),
+        };
+
+        let (content_str, media_type) =
+            match (body.content.first(), body.content.len()) {
+                (None, _) => return Ok(None),
+                (Some(first), 1) => first,
+                (_, n) => todo!("more media types than expected: {}", n),
+            };
+
+        let schema = media_type.schema.as_ref().ok_or_else(|| {
+            Error::UnexpectedFormat(
+                "No schema specified for request body".to_string(),
+            )
+        })?;
+
+        let content_type = BodyContentType::from_str(content_str)?;
+
+        let typ = match content_type {
+            BodyContentType::OctetStream => {
+                // For an octet stream, we expect a simple, specific schema:
+                // "schema": {
+                //     "type": "string",
+                //     "format": "binary"
+                // }
```

Comment:
Not sure if you meant to uncomment this code?

Reply:
No that's the JSON schema